### PR TITLE
Finished Fourth Milestone of the Redux Project

### PR DESCRIPTION
**Pull request by Psiale**
In this milestone we: 
 
- Created a Filter store
-  Created a Filter Reducer
-  Created a Filter action
- Created a Filter Component
- Displayed the Filter Component into the BooksList
- Filter books to be shown by selecting a category 
- Displayed all the books when none or "All" category is been selected

### CODE REVIEW

File: src/actions/filterActions.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { CHANGE_FILTER } from '../types/bookTypes';
+
+const changeFilter = category => {
+  // eslint-disable-next-line no-console
+  console.log('changeFilter');
```
Comment (addod19) [inline src/actions/filterActions.js]: Kindly remove console.log() statements

File: src/containers/BooksList.js [hunk 2]
```diff
@@ -3,41 +3,61 @@ import PropTypes from 'prop-types';
...
+    books, filter, removeBook, changeFilter,
+  } = props;
 
   const handleBookRemove = id => {
     removeBook(id);
   };
 
+  const handleFilterChange = event => {
+    // eslint-disable-next-line no-console
+    console.log('Am I been triggered?');
```
Comment (addod19) [inline src/containers/BooksList.js]: Kindly remove console.log() statements

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi, @Psiale and @Rhelli, great job on this project :+1: The PR is very descriptive of the work done in this milestone :fireworks: Awesome. The README file is great and we can make it even better :smile: The project works as expected :+1: However, there are a few comments we need to address before proceeding :slightly_smiling_face: KIndly check the comments below:
- `actions/index.js` file is currently empty, kindly copy the contents of `filterActions.js` into the "actions/index.js" file and delete the "filterActions" file. Also, remember to remove the console.log in line no.5
- Let us kindly rename our component; kindly rename "categoryComponent" to "categoryFilter" since all the files in the component directory are all components. Remember to change the export and import names in other files as well
- Kindly remove all debugging/console.log() in our files

[Optional]
- It would be great to improve the README file by adding a live link to the project :smile: 


Once these are fixed, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by Psiale** [comment]
Fixed changes suggested by TSE:

- Merged separated action files into actions/index.
- Changed the name of the CategoryComponent to CategoryFilter.
- Removed all console logs.
- Added live version link with the project updated.
-

**Review by shloch — APPROVED** [review]
## hello

You really have a professional approach to solving problems. Great job reacting to the above feedback. Much progress is made and the impediments are solved. Great job.
We're ready for next step. You can merge this.

HAPPY CODING

**Pull request closed without merging**